raiden_contracts/deploy/etherscan_verify.py

```diff
@@ -177,7 +178,7 @@ def etherscan_verify_contract(
         "contractname": contract_name,
         "compilerversion": "v" + metadata["compiler"]["version"],
         # 0 = Optimization used, 1 = No Optimization
-        "optimizationUsed": 0 if metadata["settings"]["optimizer"]["enabled"] is False else 1,
+        "optimizationUsed": 0 if optimization_details is not None else 1,
         "runs": metadata["settings"]["optimizer"]["runs"],
         # Typo is intentional. Etherscan does not like the correct spelling.
         "constructorArguements": constructor_args,
```

Comment:
Commit message has a typo, I think:

> but we have to test the on a testchain

(test the on)

Reply:
👀  Fixed!